src/Models/Traits/Cacheable.php

```diff
@@ -36,6 +36,7 @@ public function getCacheableProperties(): array
             'prefix' => config('cacheable.prefix', 'cacheable'),
             'identifier' => config('cacheable.identifier', 'id'),
             'logging' => [
+                'channel' => config('cacheable.logging.channel', 'default'),
```

Comment:
@Sleepy4k please default this also to `null` as `default` was removed as fallback.